### PR TITLE
Style fixes for line length, visual indent, block comments, etc.

### PR DESCRIPTION
**Pull request by rootulp**
Remove unused import

Style fix for block comments

Comment, line length, visual indent style fixes

Use four spaces for indentation instead of two

Change double quotes to single quotes for consistency

Revert four spaces to two in license

Remove 'change' and add code blocks for modification vars

### CODE REVIEW

**Comment by fzakaria** [comment]
are the changes here to make it PEP8 compliant ?

File: hypeme.py [hunk 1]
```diff
@@ -19,147 +19,144 @@
...
-      try:
-        serve_url = "http://hypem.com/serve/source/{}/{}".format(id, key)
-        request = urllib2.Request(serve_url, "" , {'Content-Type': 'application/json'})
-        request.add_header('cookie', cookie)
+
+    def __init__(self):
+        pass
+
+    def start(self):
+        print '--------STARTING DOWNLOAD--------'
```
Comment (fzakaria) [inline hypeme.py]: is this just the number of spaces that differ ?

Comment (rootulp) [inline hypeme.py]: Yep. Replaced two spaces with four spaces for indentation (from [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation))

**Comment by fzakaria** [comment]
Can you leave in this review some output of a PEP8 linter so I'm sure it passes.

I dont have time to check it myself.

**Comment by rootulp** [comment]
screen shot 2016-02-02 at 5 03 03 pm

**Comment by rootulp** [comment]
Are there any other tests you would like me to run on this PR?

**Comment by fzakaria** [comment]
No sorry, I just haven't had a chance to run it myself to verify
(been busy)
Hadn't prioritized it as its not fixing a breaking bug or anything.

Will get to it soon

Farid Zakaria

**Comment by rootulp** [comment]
Sounds good. No rush.

**Pull request merged**